_i18n/en.yml

```diff
@@ -147,42 +147,50 @@ publications-tweet_feed:
+   subtitle: <p><strong>We will help build well-crafted software that is valuable to your business.</strong></p>
+           <p>Our craftsmen have honed their skills in technologies and practices so that they deliver with quality on time. we will adapt our practices according to your needs.</p>
 services-software_creation-embedded_craftsmen:
    title: Embedded Craftsmen
    subtitle: Our craftsmen can enhance your teams to provide expertise in specific technologies. They can also form the core of your teams to help you instil XP practices and Software Craftsmanship values from the outset.
 services-software_creation-self_contained_teams:
    title: Self-contained Teams
-   subtitle: <p>Our craftsmen are equipped to cover all the roles required for a fully functioning Agile teams. We have experts in; UX and UI design, Front End and Back End Development, DevOps and Testing. All our craftsmen maintain an innate understanding of business analysis and work with the product owner to ensure that we constantly deliver value.</p>
-           <p>We will provide fully functional teams to help you get started quick and delivery you project in the most effective manner.</p>
+   subtitle: We will provide fully functional teams to help you deliver your project in the most effective and timely manner. Our craftsmen cover all the roles required for an Agile software developement team. We have experts in; UX and UI design, Mobile, Front End and Back End Development, DevOps, and Testing. 
+           <p></p>
 services-software_creation-continous_delivery:
    title: Continuous Delivery Pipeline
-   subtitle: We are experts in technologies that are enablers for Continuous Delivery. We also understand how the business needs to be aligned to realise the full benefits of this approach. We are also programatic and understand that certain organisational constrains cannot change and help you tailor the pipeline to work around these constraints.
+   subtitle: We will help you tailor a Continuous Delivery Pipeline so that your software is responsive to your needs. We are experts in technologies that are enablers for Continuous Delivery. We appreciate how the development team and business need to be aligned to realise the full potential of this approach.
 services-software_creation-ux_ui_design:
    title: UX & UI Design
-   subtitle: We have expertise in designing visual identity and user experiences and understand how this efforts fits best in an Agile environment. We will help you define the brand identity for your project and create UX and UI designs as part of your software development effort.
+   subtitle: We have expertise in designing visual identity and user experiences. We understand how this effort fits best in an Agile Softeware Deveopment environment. We will help you build the brand identity and create UX and UI designs as part of your software development effort.
```

Comment:
We have expertise in designing visual identity and user experiences. We understand how this effort fits best in an Agile Software Development environment. We will help you build the brand identity and create UX and UI designs as part of your software development effort.